diff --git a/node_modules/@graphql-mesh/utils/index.js b/node_modules/@graphql-mesh/utils/index.js
index 0e786ed..4de73f6 100644
--- a/node_modules/@graphql-mesh/utils/index.js
+++ b/node_modules/@graphql-mesh/utils/index.js
@@ -40,6 +40,11 @@ async function defaultImportFn(path) {
             return new Function(`return import(${JSON.stringify(path)})`)();
         }
         throw e;
+    }).catch(e => {
+        if (crossHelpers.path.isAbsolute(path) && !path.endsWith('.js') && !path.endsWith('.ts')) {
+        return defaultImportFn(`${path}.ts`);
+        }
+        throw e;
     });
     if (module.default != null) {
         module = module.default;
